sdk/servicebus/azure-messaging-servicebus/src/samples/java/com/azure/messaging/servicebus/ServiceBusReceiverAsyncClientRetrySample.java

```diff
@@ -0,0 +1,311 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Add link to our samples/README.md.

Reply:
thanks @conniey, both feedback addressed in the new pr https://github.com/Azure/azure-sdk-for-java/pull/29704